catkin_ws/src/calqui_msgs/action/GoToPlace.action

```diff
@@ -0,0 +1,12 @@
+#goal
+float64 x
+float64 y
+float64 theta
+---
+#result
+float64 x
+float64 y
+float64 theta
```

Comment:
the final pose of the goal is fine, but another alternative could be an enum with three states: SUCCESS/UNREACHABLE_GOAL/UNEXPECTED_FAILURE

(no need to change it, the current result is also a valid alternative)